pom.xml

```diff
@@ -28,7 +28,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.509</version>
+    <version>1.642.4</version>
```

Comment:
This change shouldn't be in the same commit/PR. But I agree to upgrade the min version of jenkins to support